docs/content/maintainers/first-issue-template.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "First Issue Template"
+date: 2018-08-25T20:00:24-07:00
+draft: false
+---
+
+This document provides a template for any issue body that is to be labeled
+`good first issue`. This should assume the contributor is either new to open
+source or at least new to the project.
+
+---
+
+### What needs to be done?
+
+Here should be a detailed outline of the issue or feature.
+
+This should include links to any relevant references both within and outside of
+the project. Other issues, pull requests or comments.
+
+Try to include ideas of what good a starting place might be.
+
+#### New To Open Source
+
```

Comment:
@robjloranger would the issue author write something about how a contributor who is new to OSS should approach this? I have the same question for the line below, about contributors who are new to the Athens project

Reply:
I'm thinking either a more general post to link, either ours or someone outside of Athens. Just like a quick start, GitHub etc. Links to our docs on continuing and PR process.